selected/selected.go

```diff
@@ -0,0 +1,6 @@
+package selected
+
+type SelectedField struct {
```

Comment:
Wouldn't it be wise to also include any arguments passed to sub-fields here?

Reply:
See https://github.com/DealTap/graphql-go/pull/5/files - though I like your approach otherwise better.

Reply:
Why do you need this?

Reply:
@pavelnikolov I'm interested in arguments to handle queries like:

```graphql
query {
  post(id: "abc") {
    # …
    comments(first: 10, order: NEWEST_FIRST) {
      # …
    }
  }
}
```

…where my GraphQL endpoint is backed by some legacy API which lets me retrieve a post along with its (newest|oldest|most popular) N comments. A list of selected fields is a start, but in this scenario I also need the arguments to know _which_ `comments` the user wants.

Arguments would be easy to add to this `SelectedField` struct so I figured that discussion can wait until this PR lands.

Reply:
Thanks for your example - it makes sense.
Yeah, let's keep this PR minimalistic and think about that later. Follow up PR would be welcome.